scripts/parser-tests/test262/allowlist.txt

```diff
@@ -6,6 +6,9 @@ language/import/import-attributes/json-named-bindings.js(default)
 language/import/import-attributes/json-named-bindings.js(strict mode)
 
 
-# 0 valid programs produced a parsing error
-
+# 4 valid programs produced a parsing error
 
+language/import/import-defer/errors/resolution-error/import-defer-of-missing-module-fails.js(default)
+language/import/import-defer/errors/resolution-error/import-defer-of-missing-module-fails.js(strict mode)
+language/import/import-defer/errors/syntax-error/import-defer-of-syntax-error-fails.js(default)
+language/import/import-defer/errors/syntax-error/import-defer-of-syntax-error-fails.js(strict mode)
```

Comment:
![image](https://github.com/user-attachments/assets/604ef2c5-f71f-4805-902d-fb35586e1396)
We can probably ignore the `resolution` error.

Reply:
Yeah, we should update our test262 runner because we only do the parsing phase.

Reply:
Oh but we are failing to _parse_ those files?

Reply:
It's a big in the tests